build_tools/python/e2e_test_framework/definitions/common_definitions.py

```diff
@@ -133,6 +133,7 @@ class Model(object):
   tags: List[str]
   source_type: ModelSourceType
   source_url: str
+  import_flags: str
```

Comment:
I'd prefer to keep `common_definitions.Model` as general as possible, while `import_flags` is IREE specific flags.

Maybe `iree_definitions.ImportedModel` is a better place for this

Reply:
Moved to ImportedModel